src/Task/Development/PackZip.php

```diff
@@ -0,0 +1,162 @@
+<?php
+namespace Robo\Task\Development;
+
+use Robo\Common\Timer;
+use Robo\Contract\PrintedInterface;
+use Robo\Result;
+use Robo\Task\BaseTask;
+use Symfony\Component\Console\Helper\ProgressBar;
+
+/**
+ * Creates a zip file.
+ *
+ * ``` php
+ * <?php
+ * $this->PackZip(
```

Comment:
there should be a trait created for this task, so you can use it from Robofile like `$this->taskPackZip`.